### PR TITLE
Add url and rpcNodeInfo to coin definition

### PR DESCRIPTION
**Pull request by hewigovens**

### CODE REVIEW

File: wallet-core/contributing.md [hunk 1]
```diff
@@ -135,22 +135,24 @@ The wallet core code generator also parses coin configuration defined in `coins.
 
 ```json
 {
-    "id": "bitcoin",                            // coin id
-    "name": "Bitcoin",                          // coin name
-    "symbol": "BTC",                            // coin symbol
-    "decimals": 8,                              // minimum currency unit, e.g. a satoshi is 0.00000001 ฿
-    "blockchain": "Bitcoin",                    // blockchain type
-    "derivationPath": "m/84'/0'/0'/0/0",        // default path for derivating private key from a mnemonic
```
Comment (vikmeup) [inline wallet-core/contributing.md]: @hewigovens I would add a mention about BIP44/SLIP44 in here, to make the developer aware of it

File: wallet-core/contributing.md [hunk 2]
```diff
@@ -135,22 +135,24 @@ The wallet core code generator also parses coin configuration defined in `coins.
...
-    "xprv": "zprv",                             // hd version bytes defined in slip-0132
-    "explorer": "https://blockchair.com/bitcoin/transaction/"// block explorer used to view transaction detail
+    "id": "bitcoin",                                            // coin id
+    "name": "Bitcoin",                                          // coin name
+    "symbol": "BTC",                                            // coin symbol
+    "decimals": 8,                                              // minimum currency unit, e.g. a satoshi is 0.00000001 ฿
+    "blockchain": "Bitcoin",                                    // blockchain type
+    "derivationPath": "m/84'/0'/0'/0/0",                        // default path for derivating private key from a mnemonic
+    "curve": "secp256k1",                                       // elliptic curve used in Bitcoin's public key cryptography
+    "publicKeyType": "secp256k1",                               // public key type
```
Comment (vikmeup) [inline wallet-core/contributing.md]: did you consider mentioning that p2pkhPrefix and other values are optional?

File: wallet-core/contributing.md (hunk 1 above)
Comment (hewigovens) [reply wallet-core/contributing.md]: yes, we will add a `slip44` field to fix https://github.com/TrustWallet/wallet-core/issues/486

File: wallet-core/contributing.md (hunk 2 above)
Comment (hewigovens) [reply wallet-core/contributing.md]: added [optional]

**Pull request merged**